addons/mail/static/tests/message/message_tests.js

```diff
@@ -1135,6 +1135,66 @@ QUnit.test("allow attachment image download on message", async () => {
     await contains(".o-mail-AttachmentImage .fa-download");
 });
 
+QUnit.test("check visibility of Download All button", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "test" });
+    const [attachmentId_1, attachmentId_2, attachmentId_3] = pyEnv["ir.attachment"].create([
+        { name: "test.png", mimetype: "image/png" },
+        { name: "Blah.png", mimetype: "image/png" },
+        { name: "shut.png", mimetype: "image/png" },
+    ]);
+    pyEnv["mail.message"].create([{
+        attachment_ids: [attachmentId_1, attachmentId_2],
+        body: "<p>Test</p>",
+        model: "discuss.channel",
+        res_id: channelId,
+        message_type: "comment",
+    }, {
+        attachment_ids: [attachmentId_3],
+        model: "discuss.channel",
+        res_id: channelId,
+        message_type: "comment",
+    }]);
+    const { openDiscuss } = await start();
+    await openDiscuss(channelId);
+    await click("[title='Expand']:eq(0)");
+    await contains(".o-dropdown--menu span[title='Download All']");
+    await click("[title='Expand']:eq(1)");
+    await contains(".o-dropdown--menu span[title='Download All']", { count: 0 });
```

Comment:
`.o-dropdown--menu span` is unnecessary. Just `[title='Download Files']` is good enough.
Shorter makes it easier to read, understand, maintain code in test